services/QuillLMS/engines/evidence/app/services/evidence/gen_ai/repeated_feedback/prompt_builder.rb

```diff
@@ -3,14 +3,39 @@
 module Evidence
   module GenAi
     module RepeatedFeedback
-      class PromptBuilder < Evidence::GenAi::PromptBuilder
-        TEMPLATE_FOLDER = 'app/services/evidence/gen_ai/repeated_feedback/prompts/'
-        DEFAULT_TEMPLATE = '2024_08_28_gemini2.md'
+      class PromptBuilder < ApplicationService
+        include Evidence::MarkdownFormatter
 
-        private def template_variables = { feedback_history: }
+        # Copied from prompts/2025_07_14_gemini-flash-2.0-2.md
```

Comment:
I'm curious why you switched from TEMPLATE_FOLDER and DEFAULT_TEMPLATE pattern here to hard coding within class.